### ISSUE TITLE
[BUG] Error running performance analysis in telegram

### ISSUE THREAD

**Issue by znerol19619** [CoinAlpha/hummingbot#1069]
**Describe the bug**
Error running ``history`` command in telegram. 

**Steps To Reproduce**
Integrate hummingbot first with your telegram
Run pmm for testing and wait for an order to be filled, then run history command in telegram. Tho there are times where error still occurs even without any filled orders. 

**Screenshots**
![Screenshot from 2019-11-15 17-46-26](https://user-images.githubusercontent.com/27669951/68933605-f5b31380-07cf-11ea-92bd-af4276ae6ff0.png)

**Error**
```
2019-11-15 08:39:41,482 - 11 - hummingbot.client.hummingbot_application - ERROR - Unexpected error running performance analysis.
Traceback (most recent call last):
  File "/opt/conda/envs/hummingbot/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1179, in _execute_context
    context = constructor(dialect, self, conn, *args)
  File "/opt/conda/envs/hummingbot/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 686, in _init_compiled
    self.cursor = self.create_cursor()
  File "/opt/conda/envs/hummingbot/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 1038, in create_cursor
    return self._dbapi_connection.cursor()
  File "/opt/conda/envs/hummingbot/lib/python3.6/site-packages/sqlalchemy/pool/base.py", line 965, in cursor
    return self.connection.cursor(*args, **kwargs)
sqlite3.ProgrammingError: SQLite objects created in a thread can only be used in that same thread. The object was created in thread id 140281340761472 and this is thread id 140278455330560
```

**Release version**
``ver-0.18.1`` &

```
 "branch": "development",
 "commit": "99a6e4d12aa84bd067ae6b2eeb5d62e688d730e0",
 "date": "2019-11-15T00:23:26Z",
```

**Attachments**
[logs_conf_pure_market_making_strategy_0.txt](https://github.com/CoinAlpha/hummingbot/files/3850850/logs_conf_pure_market_making_strategy_0.txt)

**Optional: your discord username:** newcryptodev
**Optional: your ETH address:** (If you want to participate in the bug bounty)

**Comment by MishaelTereso** [comment]
![tempsnip_telegram_bot](https://user-images.githubusercontent.com/56957119/69031953-46b44900-0a16-11ea-9f09-cf932078448a.png)

![tempsnip_telegram2_bot](https://user-images.githubusercontent.com/56957119/69032676-081f8e00-0a18-11ea-9271-4daebfb7b818.png)


1. Updated dev branch with the latest commits - https://github.com/CoinAlpha/hummingbot/pull/1072/commits
2. Integrated Hummingbot on my Telegram
3. Ran PMM, order created (not yet filled)
4. Entered /history command on Telegram. Hummingbot unable to recognize the command.
5. Entered /history and /status command on Hummingbot, Telegram unable to recognize the command.
6. Different errors are showing.

**Comment by znerol19619** [comment]
Tested this branch https://github.com/CoinAlpha/hummingbot/pull/1072/commits .

I've been running it for 2 hrs, so far no errors are showing performing ``history`` command both in telegram and hummingbot.

**Screenshots**

![Screenshot from 2019-11-18 15-36-05](https://user-images.githubusercontent.com/27669951/69035833-859acc80-0a1f-11ea-8bd8-c49b8340d3c3.png)
![Screenshot from 2019-11-18 15-35-26](https://user-images.githubusercontent.com/27669951/69035823-816eaf00-0a1f-11ea-8a42-1edc34be437d.png)
![Screenshot from 2019-11-18 15-35-45](https://user-images.githubusercontent.com/27669951/69035826-83387280-0a1f-11ea-9ef3-0f7882df08dd.png)

**Comment by PtrckM** [comment]
@znerol19619  confirmed fix on 0.19.0

**Comment by znerol19619** [comment]
Confirmed working. I will now close this bug report.

```
                "branch": "development",
                "commit": "59a64224020e8f598d5612a784c27c0d7fc16bf2",
                "date": "2019-11-22T20:22:41Z",
```
